src/software/simulation/standalone_simulator_main.cpp

```diff
@@ -1,24 +1,52 @@
 #include "software/gui/standalone_simulator/threaded_standalone_simulator_gui.h"
 #include "software/logger/logger.h"
-#include "software/parameter/dynamic_parameters.h"
 #include "software/simulation/standalone_simulator.h"
+#include "software/world/field.h"
 
-int main()
+int main(int argc, char *argv[])
 {
     LoggerSingleton::initializeLogger();
 
-    ThreadedStandaloneSimulatorGUI standalone_simulator_gui_wrapper;
     StandaloneSimulator standalone_simulator(
         MutableDynamicParameters->getMutableStandaloneSimulatorConfig());
+    standalone_simulator.setupInitialSimulationState();
+
+    auto ball_placement_callback = [&standalone_simulator](Point ball_placement_point) {
+        BallState state(ball_placement_point, Vector(0, 0));
+        standalone_simulator.setBallState(state);
+    };
+
+    auto simulation_mode_callback =
+        [&standalone_simulator](StandaloneSimulator::SimulationMode new_simulation_mode) {
+            switch (new_simulation_mode)
+            {
+                case StandaloneSimulator::SimulationMode::PLAY:
+                    standalone_simulator.resetSlowMotionMultiplier();
+                    standalone_simulator.startSimulation();
```

Comment:
why do we start the simulation here? isn't it already running?

Reply:
Not necessarily, could be paused.